.cog/veneers/dashboard.common.yaml

```diff
@@ -25,6 +25,140 @@ builders:
   - omit: { by_object: DynamicConfigValue }
   - omit: { by_object: DataTransformerConfig }
 
+  #######################
+  # Overrides shortcuts #
+  #######################
+
+  # byName
+  - add_option:
+      by_object: FieldConfigSource
+      option:
+        name: overrideByName
+        comments:
+          - Adds override rules for a specific field, referred to by its name.
+        arguments:
+          - name: name
+            type: &type_string
+              kind: scalar
+              scalar: {scalar_kind: string}
+          - name: properties
+            type: &type_DynamicConfigValue_array
```

Comment:
I don't understand this...

Reply:
This syntax basically allows the definition of an alias, referencing the value being defined (here: an array definition)

The alias can then be used later with the `*alias_name` syntax. It's basically a way to copy-paste stuff without actually copy-pasting it :sweat_smile: